covid-vue-app/src/components/activeCasesOverview.vue

```diff
@@ -1,30 +1,64 @@
 <template lang="html">
-  <v-card>
-    <v-card-title>Active Cases</v-card-title>
+  <v-card style="background: #4da6ff;">
     <v-row>
       <v-col align="center">
-        <p class="headline">Currently infected : {{ stats.cases }}</p>
-        <p class="headline">Mild condition : {{ stats.mild }}</p>
-        <p class="headline">Serious or Critical : {{ stats.critical }}</p>
+        <h3 style="color:white;" class="display-1 py-6"><b>Active Cases in Ireland</b></h3>
+        <p style="color:white;" class="headline">Currently infected : {{ stats.infected }}</p>
+        <p style="color:white;" class="headline">Last Checked : {{ stats.checked }}</p>
+        <p style="color:white;" class="headline">Last Reported : {{ stats.reported }}</p>
+        <v-btn @click="getData">Get Data</v-btn>
       </v-col>
     </v-row>
   </v-card>
 </template>
 
 <script>
+import axios from "axios";
```

Comment:
Can't be sure about `Vue` but I reckon we could separate the `Javascript` and the `structure`

Reply:
Nvm, this was the only `separation of concerns` for the traditional web development, whereas we are *supposed* to contain the structure and logic for individual components.